client/packages/common/src/ui/forms/Detail/NumericTextDisplay.tsx

```diff
@@ -16,17 +16,19 @@ interface NumericTextDisplayProps {
   children?: React.ReactElement;
   width?: number | string;
   packagingDisplay?: string;
+  decimalLimit?: number;
 }
 
 export const NumericTextDisplay: FC<NumericTextDisplayProps> = ({
   value,
   defaultValue = '',
   width = 50,
   packagingDisplay,
+  decimalLimit = 2,
```

Comment:
we used to default to no decimals. 2.10 added 2dp as a default, but this shouldn't apply in requisitions (at the ordering level we only would work in whole units) so we need to override